packages/animations/example/lib/container_transition.dart

```diff
@@ -0,0 +1,419 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+
+const String _loremIpsumParagraph = '''
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Vulputate dignissim suspendisse in est. Ut ornare lectus sit amet. Eget nunc lobortis mattis aliquam faucibus purus in. Hendrerit gravida rutrum quisque non tellus orci ac auctor. Mattis aliquam faucibus purus in massa. Tellus rutrum tellus pellentesque eu tincidunt tortor. Nunc eget lorem dolor sed. Nulla at volutpat diam ut venenatis tellus in metus. Tellus cras adipiscing enim eu turpis. Pretium fusce id velit ut tortor. Adipiscing enim eu turpis egestas pretium. Quis varius quam quisque id. Blandit aliquam etiam erat velit scelerisque. In nisl nisi scelerisque eu. Semper risus in hendrerit gravida rutrum quisque. Suspendisse in est ante in nibh mauris cursus mattis molestie. Adipiscing elit duis tristique sollicitudin nibh sit amet commodo nulla. Pretium viverra suspendisse potenti nullam ac tortor vitae.
+''';
+
+/// The demo page for [OpenContainerTransform].
+class OpenContainerTransformDemo extends StatefulWidget {
+  @override
+  _OpenContainerTransformDemoState createState() =>
+      _OpenContainerTransformDemoState();
+}
+
+class _OpenContainerTransformDemoState
+    extends State<OpenContainerTransformDemo> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Container Transform')),
+      body: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: ListView(
+          children: <Widget>[
+            _OpenContainerWrapper(
+              closedBuilder:
+                  (BuildContext context, VoidCallback openContainer) {
+                return _ExampleCard(openContainer: openContainer);
```

Comment:
If you want to use the openContainer callback, you should probably set `tappable` to false on the `OpenContainer` widget (or just not use the openContainer callback and let the widget handle it).

Reply:
In most cases, using the openContainer callback made more sense because we would like to see ink splash effects. I think using just the default `tappable: true` does not achieve this

Reply:
I'll make changes to set tappable to false where appropriate!